util/abs/provision_pe_xl_nodes.rb

```diff
@@ -8,6 +8,8 @@
 require "./setup/helpers/abs_helper.rb"
 include AbsHelper # rubocop:disable Style/MixinUsage
 
+ARCH = { large: "l", extra_large: "xl" }.freeze
```

Comment:
We often refer to it as the ref arch.  So maybe make this REF_ARCH_TYPES  and DEFAULT_REF_ARCH

Reply:
:+1: I have updated to change the variables and option accordingly.